<launch>

  <node pkg="rollout_t42" name="rollout_recorder" type="rollout_recorder.py" output="screen"/>

  <node pkg="cl_control" name="general_control" type="general_control.py" output="screen"/>
  <!-- <node pkg="cl_control" name="gp_controller" type="gp_controller.py" output="screen"/> -->
  <node pkg="cl_control" name="vs_controller" type="vs_controller.py" output="screen"/>

  <node pkg="cl_control" name="plot_obj_pos" type="plot_obj_pos.py" output="screen"/>


</launch>